context/app/static/js/components/Detail/Summary/Summary.spec.js

```diff
@@ -22,7 +16,7 @@ test('timestamps display when defined', () => {
   const textToTest = ['Creation Date', 'Modification Date'];
   textToTest.forEach((text) => expect(screen.getByText(text)).toBeInTheDocument());
 
-  expect(screen.getAllByText('August 06, 2020')).toHaveLength(2);
+  expect(screen.getAllByText('2020-08-06')).toHaveLength(2);
```

Comment:
I don't know the history, but I think the format with the month spelled out is a feature we don't want to lose in the shuffle.

Reply:
This was changed in the design.